src/fides/api/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -172,7 +172,8 @@ def get_request_origin_and_config() -> Tuple[Optional[str], Optional[str]]:
         return origin, experience_config_history_identifier
 
     request_headers = request.headers
-    ip_address: Optional[str] = request.client.host if request.client else None
+
+    ip_address: Optional[str] = get_ip_address(request)
```

Comment:
Note Served Notice History endpoints are in  OSS but Privacy Preference History endpoints are in Plus.

They share this IP address code so if it works here, it should work over in Plus.